trollflow2/tests/test_trollflow2.py

```diff
@@ -1554,7 +1554,8 @@ def test_discard_old_data(self):
             job = {'product_list': None, 'input_mda': {'start_time': dt.datetime(2020, 3, 18)}}
             self.assertIsNone(check_metadata(job))
             get_config_value.return_value = {'start_time': -2e6}
-            self.assertIsNone(check_metadata(job))
+            with self.assertRaises(AbortProcessing):
+                check_metadata(job)
```

Comment:
I don't think that's the right solution, and I actually already fixed this in https://github.com/pytroll/trollflow2/pull/199/commits/99d4a6b9a6bbad8f04dc7874a56b393c0eea5dbf
(https://github.com/pytroll/trollflow2/pull/199 waited for a positive review to be auto merged, but realise it was only a comment. merging now)

Reply:
Fixed by rebasing with current `main`.